telethon/events/callbackquery.py

```diff
@@ -118,8 +118,10 @@ def filter(self, event):
             elif event.query.data != self.match:
                 return
 
-        if not self.func or self.func(event):
-            return event
+        if self.func:
```

Comment:
A `return not self.func or self.func(event)` would've actually worked but there's no need to change it back, this one is more readable.